packages/buckets/src/api/index.ts

```diff
@@ -617,17 +647,20 @@ export async function bucketsPushPath(
       client.send(req)
 
       if (source.content) {
-        const process = await block({ size: 32768, noPad: true })
-        for await (const chunk of process(source.content)) {
+        for await (const chunk of source.content) {
           // Let's just make sure we haven't aborted this outside this function
           if (opts?.signal?.aborted) {
-            client.close()
+            try {
+              client.close()
+            } catch {} // noop
             return reject(AbortError)
           }
-          const buf = chunk.slice()
-          const part = new PushPathRequest()
-          part.setChunk(buf as Buffer)
-          client.send(part)
+          // Naively chunk into chunks smaller than CHUNK_SIZE bytes
```

Comment:
_Now_ we try to chunk it up into parts of roughly CHUNK_SIZE, which may or may not be exactly what we want, but should be close.